docs/Tutorials/CCE.md

```diff
@@ -134,61 +134,22 @@ The `EthInertialRetardedTime` is a diagnostic dataset that represents the
 angular derivative of the inertial retarded time, which determines the
 coordinate transformation that is performed at scri+.
 
-The following python script will load data from a successful CCE run and
-construct a plot of all of the physical waveform data.
-```py
-import matplotlib.pyplot as plt
-import numpy as np
-import h5py as h5
+If you'd like to visualize the output of a CCE run, we offer a
+[CLI](py/cli.html) that will produce a plot of all of quantities except
+`EthInertialRetardedTime`. To see how to use this cli, run
 
+```
+spectre plot cce -h
+```
 
-def spectre_real_mode_index(l, m):
-    return 2 * (l**2 + l + m) + 1
-
-
-def spectre_imag_mode_index(l, m):
-    return 2 * (l**2 + l + m) + 2
-
-
-def get_modes_from_block_output(filename, dataset, modes=[[2, 2], [3, 3]]):
-    with h5.File(filename, "r") as h5_file:
-        timeseries_data = (h5_file[dataset][()][:, [0] + list(
-            np.array([[
-                spectre_real_mode_index(x[0], x[1]),
-                spectre_imag_mode_index(x[0], x[1])
-            ] for x in modes]).flatten())])
-    return timeseries_data
-
-
-plot_quantities = ["Strain", "News", "Psi0", "Psi1", "Psi2", "Psi3", "Psi4"]
-mode_set = [[2, 2], [3, 3]]
-filename = "CharacteristicExtractReduction.h5"
-output_plot_filename = "CCE_plot.pdf"
-
-legend = []
-for (mode_l, mode_m) in mode_set:
-    legend = np.append(legend, [
-        r"Re $Y_{" + str(mode_l) + r"\," + str(mode_m) + r"}$",
-        r"Im $Y_{" + str(mode_l) + r"\," + str(mode_m) + r"}$"
-    ])
+If you'd like to do something more complex than just make a quick plot, you'll
```

Comment:
maybe `complex` -> `complicated`? Since CCE deals with complex numbers, just to avoid any possible confusion? Up to you!